tests/third_party/json-c/math_compat.h

```diff
@@ -19,12 +19,16 @@
 
 #ifndef HAVE_DECL_INFINITY
 #include <float.h>
+#ifndef INFINITY
```

Comment:
We should try to avoid carrying local changes for json-c, if this is really needed then lets get it merged upstream first.